modules/formulize/rest/index.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+###############################################################################
+##     Formulize - ad hoc form creation and reporting module for XOOPS       ##
+##                    Copyright (c) Formulize Project                        ##
+###############################################################################
+##                    XOOPS - PHP Content Management System                  ##
+##                       Copyright (c) 2000 XOOPS.org                        ##
+##                          <http://www.xoops.org/>                          ##
+###############################################################################
+##  This program is free software; you can redistribute it and/or modify     ##
+##  it under the terms of the GNU General Public License as published by     ##
+##  the Free Software Foundation; either version 2 of the License, or        ##
+##  (at your option) any later version.                                      ##
+##                                                                           ##
+##  You may not change or alter any portion of this comment or credits       ##
+##  of supporting developers from this source code or any supporting         ##
+##  source code which is considered copyrighted (c) material of the          ##
+##  original comment or credit authors.                                      ##
+##                                                                           ##
+##  This program is distributed in the hope that it will be useful,          ##
+##  but WITHOUT ANY WARRANTY; without even the implied warranty of           ##
+##  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the            ##
+##  GNU General Public License for more details.                             ##
+##                                                                           ##
+##  You should have received a copy of the GNU General Public License        ##
+##  along with this program; if not, write to the Free Software              ##
+##  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307 USA ##
+###############################################################################
+##  Author of this file: Formulize Project                                   ##
+##  Project: Formulize                                                       ##
+###############################################################################
+
+// handle rest api requests, sent to /formulize-rest-api/
+// URL syntax should be:
+// /formulize-rest-api/{version}/{object-or-action}
+
+// supported actions are:
+// ping - responds with a 1, to show the API is active/available
+// queue/{id} - triggers parsing of the queue specified, if no {id} then runs all queues
```

Comment:
Am I missing something. I don't see a file for the queue operation in this PR? Additionally when introducing RPC operations such as the processing of a queue I would recommend following the structure `/api/queue/{id}/process`. From a restful perspective it's more clear what the action is. Performing a POST on the ID it's not clear what the outcome would be.

Reply:
It's coming. Good point about /process. A PUT on queue/{id} would update the queue. But more probably a POST to /formulize-public-api/v1/queue/{id}/add would stick something at the end of the queue?